pycaret/internal/tabular.py

```diff
@@ -3257,21 +3287,68 @@ def create_model_supervised(
             logger.info("Finalizing model")
             with io.capture_output():
                 pipeline_with_model.fit(data_X, data_y, **fit_kwargs)
+            
+            if isinstance(model, CustomProbabilityThresholdClassifier):
```

Comment:
Could you add some notes here as to what is happening here? It will help to review this.

Reply:
Yes, I would add some notes here.